lib/internal/validators.js

```diff
@@ -152,28 +227,36 @@ function getOwnPropertyValueOrDefault(options, key, defaultValue) {
 }
 
 /**
- * @param {unknown} value
+ * @callback validateObject
+ * @param {*} value
  * @param {string} name
- * @param {{
- *   allowArray?: boolean,
- *   allowFunction?: boolean,
- *   nullable?: boolean
- * }} [options]
+ * @param {{allowArray: boolean=, allowFunction: boolean=, nullable: boolean=}} [options]
```

Comment:
Also seems unrelated, let's revert.
```suggestion
 * @param {{
 *   allowArray?: boolean,
 *   allowFunction?: boolean,
 *   nullable?: boolean
 * }} [options]
```